src/content/docs/apm/agents/java-agent/frameworks/scala-akka-http-core.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Scala Akka HTTP core instrumentation
+tags:
+  - Agents
+  - Java agent
+  - Scala instrumentation
+  - Custom instrumentation
```

Comment:
Not sure, but this tag may not be needed. This change doesn't involve any custom instrumentation from the user

Reply:
Understood; these are doc related tags used in searching, so I think we're ok